manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  default_env:
+    bosh:
+      password: (( grab secrets.vcap_password ))
+
+vm_types:
+  - name: small
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
```

Comment:
Indeed our small are the same as mediums, but I think it's good to keep different references, as that indicates how much resources the given job really needs. We could also completely review and optimise this, however due to nature of T2 we probably can't switch too many VMs to it.